src/Backend/Core/Engine/Meta.php

```diff
@@ -477,6 +477,11 @@ public function save($update = false)
             $custom = null;
         }
 
+        //serialize data for save
+        if (!empty($this->data['data'])) {
+            $this->data['data'] = serialize($this->data['data']);
```

Comment:
The problem is that you start from your own master branch and add your fixes to it, it is better to create a seperate branch for each pull request you want to make and keep your master in sync with the master of fork